From ac7a57a3b10708fa0a2ed0aef293866608500326 Mon Sep 17 00:00:00 2001
From: Alexandre Montplaisir <alexandre.montplaisir@polymtl.ca>
Date: Tue, 24 May 2011 18:32:51 -0400
Subject: [PATCH 137/757] Specify the complete paths to define_trace.h

The "make" command used by DKMS does not cd into every directory like
"make modules" seems to do. This patch allows compilation for both
methods.

Signed-off-by: Alexandre Montplaisir <alexandre.montplaisir@polymtl.ca>
Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/instrumentation/events/lttng-module/block.h    | 2 +-
 drivers/staging/lttng/instrumentation/events/lttng-module/irq.h      | 2 +-
 drivers/staging/lttng/instrumentation/events/lttng-module/kvm.h      | 2 +-
 drivers/staging/lttng/instrumentation/events/lttng-module/lttng.h    | 2 +-
 drivers/staging/lttng/instrumentation/events/lttng-module/sched.h    | 2 +-
 drivers/staging/lttng/instrumentation/events/lttng-module/syscalls.h | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/block.h b/drivers/staging/lttng/instrumentation/events/lttng-module/block.h
index 1c48384..5ecc95a 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/block.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/block.h
@@ -616,5 +616,5 @@ TRACE_EVENT(block_rq_remap,
 #endif /* _TRACE_BLOCK_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
 
diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/irq.h b/drivers/staging/lttng/instrumentation/events/lttng-module/irq.h
index ac590c9..344015d 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/irq.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/irq.h
@@ -152,4 +152,4 @@ DEFINE_EVENT(softirq, softirq_raise,
 #endif /*  _TRACE_IRQ_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/kvm.h b/drivers/staging/lttng/instrumentation/events/lttng-module/kvm.h
index 43914b3..e10455b 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/kvm.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/kvm.h
@@ -309,4 +309,4 @@ TRACE_EVENT(
 #endif /* _TRACE_KVM_MAIN_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/lttng.h b/drivers/staging/lttng/instrumentation/events/lttng-module/lttng.h
index 4cc14fc..6f3d6d1 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/lttng.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/lttng.h
@@ -31,4 +31,4 @@ TRACE_EVENT(lttng_metadata,
 #endif /*  _TRACE_LTTNG_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/sched.h b/drivers/staging/lttng/instrumentation/events/lttng-module/sched.h
index 37debc3..fd78f24 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/sched.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/sched.h
@@ -397,4 +397,4 @@ TRACE_EVENT(sched_pi_setprio,
 #endif /* _TRACE_SCHED_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/syscalls.h b/drivers/staging/lttng/instrumentation/events/lttng-module/syscalls.h
index 7392e98..e2f3bbd 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/syscalls.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/syscalls.h
@@ -69,5 +69,5 @@ TRACE_EVENT(sys_exit,
 #endif /* _TRACE_EVENTS_SYSCALLS_H */
 
 /* This part must be outside protection */
-#include "define_trace.h"
+#include "../../../probes/define_trace.h"
 
-- 
2.0.2

